hexhttp.py

```diff
@@ -273,6 +321,8 @@ def main(urli, s):
     user_agent = results.user_agent
     threads = results.threads
 
+    configure_logging(results.verbose, results.log, results.log_file)
```

Comment:
use the verbose, log and logfile from the command line arguments to configure the logging module